src/app/modules/angular-slickgrid/components/angular-slickgrid.component.ts

```diff
@@ -244,9 +244,8 @@ export class AngularSlickgridComponent implements AfterViewInit, OnDestroy {
           this.filterService.refreshTreeDataFilters();
         }
       });
+      this._isDatasetHierarchicalInitialized = true;
     }
-
-    this._isDatasetHierarchicalInitialized = true;
   }
```

Comment:
adding `this.grid.invalidate();` in Aurelia helps to avoid flickering with initial tree hierarchical tree collapse